pkg/runtime/conversion_generator.go

```diff
@@ -302,11 +302,15 @@ func (g *conversionGenerator) generateConversionsForSlice(inType, outType reflec
 }
 
 func (g *conversionGenerator) generateConversionsForStruct(inType, outType reflect.Type) error {
+	errs := []string{}
```

Comment:
>
generateConversionsBetween is not a no-op. Calling it for all possible fields in the struct ensures conversion functions exist for those types, whether or not an automated conversion can be created for this struct.

Yes - but what matter in the end is whether you return an error or nil. And this is not changed in this PR.